Update Beta changelog (#463)
CloudCannon/platform-documentation: articles/beta-changelog.mdx

* Clicking on a domain card in the *Domains browser* will also open that domain's *Domain* page.
* The *Domain* page allows you to review and manage all settings for a given domain.
* The *Domain* page has four tabs: the *Network* tab, the *DNS* tab, the *Sites* tab, and the *Domain Settings* tab.
  * The *Network* tab shows the status of your Nameservers, lists all your subdomains in a *Subdomains browser*, and shows the Fallback redirect for all unconfigured subdomains.
  * The *DNS* tab lists all DNS records configured for your domain, and includes a search bar and *Create record* button in the top right. You can edit or delete each DNS record using the *Context Menu* in the top right of the card.
  * The *Sites* tab lists of all Sites connected to your domain. You can navigate to a Site by clicking on the card.
  * The *Domain Settings* tab has two sections: *Details* and *Danger*. You can edit your *DNS Provider* or release the domain from your Organization.

> * You cannot release a domain from your Organization if it is connected to any Sites.

Reviewer: ```suggestion
* You cannot disconnect a domain from your Organization if it is connected to any Sites.
```

* You can now select the domain for the Fallback redirect using a dropdown, rather than a list.

<comp.DocsImage path="https://cc-dam.imgix.net/documentation/images/change-log/2025-03/CloudCannon-Changelog-Domain-Page-Network-Subdomain-Tab.png" alt="A screenshot of the Subdomains section of the Network tab shows the Hosting Setup, SSL, and Site browser." title="The Domain page" type="screenshot"/>

* Clicking on a subdomain card in the *Subdomains browser* will open the *Subdomains* section of the *Network* tab.
* The *Subdomains* section allows you to configure your Hosting Setup and SSL, and also has a Sites browser for all Sites connected to that subdomain.

<comp.DocsImage path="https://cc-dam.imgix.net/documentation/images/change-log/2025-03/CloudCannon-Changelog-Site-Settings-Custom-Domain-Page-Empty.png" alt="A screenshot of the Custom Domain page shows the Base Domain dropdown, the Domain Name and Subpath text fields, and the Add Domain button." title="The Custom Domain page" type="screenshot"/>

* The *Custom Domain* page under *Site Settings* no longer shows your hosting setup, nameserver status, or SSL settings.
  * These settings are now on your Domain page.

> * If you don't have a domain connected to your Site, the Custom Domain page will allow you to add an existing domain from your Organization, or create a domain using the *Base Domain* dropdown.

Reviewer: ```suggestion
* If you don't have a domain added to your Site, the Custom Domain page will allow you to add an existing domain from your Organization, or create a domain using the *Base Domain* dropdown.
```

* Updated messaging on the *Build Pending* page.

## Fixes

* Fixed an issue where, when configuring Snippets, CloudCannon incorrectly required you to define the `view` key within the `preview` object. You now configure the `view` key directly inside the Snippet.
* Fixed an issue where, when you selected a new method on the *Authentication* page, CloudCannon would display the inputs for a new Authentication method incorrectly before you confirmed your choice.
* Fixed an issue where Color inputs would incorrectly display the color preview for HSV colors.
* Fixed an issue where you could not close the color picker in a Color input by pressing the tab key.
* Fixed an issue where, in some cases, CloudCannon would incorrectly enable alpha for Color inputs.
* Fixed an issue where CloudCannon failed to convert existing color values to the correct format.
* Fixed an issue where, in some cases, you could not clear the value of an Image input in the Content Editor.

> * Fixed an issue where the gallery preview on an Image snippet in the Content Editor would disappear on click or drag.

Reviewer: ```suggestion
* Fixed an issue where the gallery preview on an Image in rich text would disappear on click or drag.
```
